src/rhsmlib/services/entitlement.py

```diff
@@ -58,12 +57,6 @@ def parse_date(cls, on_date: str) -> datetime.datetime:
         return on_date
 
     def get_status(self, on_date: str = None, force: bool = False) -> dict:
```

Comment:
Argument `on_date` is not used. Please, remove it and do refactoring of usage of this method.